### ISSUE TITLE
WalletConnect deep-link doesn't work on Windows

### ISSUE THREAD

**Issue by nop33** [alephium/desktop-wallet#670]
```
index-f336ecee.js:565 Uncaught (in promise) SyntaxError: Unexpected end of data (at index-f336ecee.js:565:55464)
    at YLe (index-f336ecee.js:565:55464)
    at WLe.decode [as baseDecode] (index-f336ecee.js:565:55833)
    at WLe.decode (index-f336ecee.js:565:54242)
    at Ws (index-f336ecee.js:565:61352)
    at QQ (index-f336ecee.js:568:999)
    at AFe.decode (index-f336ecee.js:574:36629)
    at Yn.<anonymous> (index-f336ecee.js:574:64706)
    at Yn.emit (index-f336ecee.js:137:97032)
    at NFe.onMessageEvent (index-f336ecee.js:574:54149)
    at async NFe.onProviderPayload (index-f336ecee.js:574:54042)
```

**Comment by mvaivre** [comment]
FWIW, [here's](https://github.com/iotaledger/firefly/blob/ffc85d1a2ea3d51313c738755b660947141b609e/packages/desktop/electron/main.js#L466-L508) how Firefly does deep linking.

**Comment by nop33** [comment]
FWIW, the firefly code related to deep linking starts a bit earlier, this is the windows-related part: https://github.com/iotaledger/firefly/blob/ffc85d1a2ea3d51313c738755b660947141b609e/packages/desktop/electron/main.js#L450-L464

**Comment by nop33** [comment]
merged in v2.0. Closing.
